src/routes/RemoteMic/Panels/RemoteSongList/LanguageFilter.tsx

```diff
@@ -32,11 +32,16 @@ export default function LanguageFilter({ children, languageList, excludedLanguag
     <>
       {open && (
         <Modal onClose={() => setOpen(false)}>
-          <MenuContainer className="!gap-1">
+          <MenuContainer data-test="languages-container" className="!gap-1">
             {languageList
               .filter((lang) => lang.count > MIN_SONGS_COUNT)
               .map(({ name, count }) => (
-                <MenuButton size={'small'} key={name} onClick={() => excludeLanguage(name)}>
+                <MenuButton
+                  size={'small'}
+                  key={name}
+                  data-active={excludedLanguages.length && !excludedLanguages.includes(name)}
```

Comment:
you should put `!!` before `excludedLanguages.length` to avoid having the attribute become 0 when no language is excluded.

Better yet, create a new variable with the expression before `return` statement and use it here nad in the `className` prop, e.g.:

```ts
const isActive = !!excludedLanguages.length && !excludedLanguages.includes(name);


return ( // [...]
data-active={isActive}
// [...]
className={isActive ? 'text-active' : ''}
// [...]
);
```

Reply:
I just added '!!' in the mentioned line without creating variables (there were some issues with the suggested changes)